CHANGES

```diff
@@ -1,6 +1,7 @@
 214
 - Fix log file rename in launcher script.
 - Remove Executed class from bootstrap module.
+- Add "bootstrap.quiet" configuration property.
```

Comment:
What does it do?

Also, will it help with @nineinchnick's https://github.com/trinodb/trino/pull/10689?

Reply:
It would help with that. The intent is to set it in `DistributedQueryRunner` without needing to touch any connectors.